interpreters/python/Makefile

```diff
@@ -0,0 +1,165 @@
+############################################################################
+# apps/interpreters/python/Makefile
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+include $(APPDIR)/Make.defs
+
+CPYTHON_URL ?= "https://github.com/python/cpython/archive"
+CPYTHON_VERSION = $(patsubst "%",%,$(strip $(CONFIG_INTERPRETER_CPYTHON_VERSION)))
+CPYTHON_ZIP = v$(CPYTHON_VERSION).zip
+
+CPYTHON_UNPACKNAME = Python
+UNPACK ?= unzip -q -o
+
+MACHDEP=nuttx
+CONFIG_SITE=${CURDIR}/config.site
+CPYTHON_PATH=$(CURDIR)/$(CPYTHON_UNPACKNAME)
+
+BUILDIR=$(CURDIR)/build
+INSTALLDIR=$(CURDIR)/install
+HOSTBUILD=$(BUILDIR)/host
+HOSTINSTALL=$(INSTALLDIR)/host
+HOSTPYTHON=$(HOSTINSTALL)/bin/python3
+TARGETBUILD=$(BUILDIR)/target
+TARGETINSTALL=$(INSTALLDIR)/target
+TARGETLIBPYTHON=$(TARGETINSTALL)/libpython3.13.a
+TARGETMODULESPACK=$(TARGETBUILD)/lib/python313.zip
```

Comment:
why download use config but here hardcode version number?

Reply:
Fixing it...